packages/web/tsconfig.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
should we also have a tsconfig on root?

Reply:
if other packages are going to use ts too, it is good to have a common on root and extend on packages as needed.

Reply:
Good point, there is already a `tsconfig.json` on root, I will look for a way to extend it

Reply:
I deleted all the other options that are set on the root `tsconfig.son` or not set at all. I think it makes easier to see what changes between each tsconfig file.